datareduction/datareductionoperator.cpp

```diff
@@ -2015,7 +2015,7 @@ namespace DRO {
    // Accumulate contributions coming from this velocity block to the
    // spatial cell velocity moments. If multithreading / OpenMP is used,
    // these updates need to be atomic:
-#pragma omp critical
+///#pragma omp critical
          { epsilon += thread_epsilon_sum; }
```

Comment:
Opinions @ursg @ykempf - do we want to remove the commented out omp pragmas from this file? And the critical region curlies? If they are all going to be handled by ARCH anyway.

Reply:
In principle, yes. If it works, and guarantees consistency and all.
Should we leave a friendly warning comment there though? For the future?

Reply:
I didn't do this yet, maybe if you have an idea about what kind of warnings to keep, please feel free to add commits (commits allowed for maintainers checked).